### PR TITLE
add and delete functionality

### PR DESCRIPTION
**Pull request by AbbyNyakara**
- The to do list should take the user's input. The user should be able to add an item and delete an item.
- The icons should change when the user clicks
- Added functionality to make the text editable
- Remove the hard coded items 
- Implement the local storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,42 +1,108 @@
...
+      const deleteTask = element.parentNode.parentNode;
+      trashIcon.onclick = () => {
+        listArray[element.id].trash = true;
+        listArray[element.id].completed = true;
+        deleteTask.remove();
+      };
+      // Filter the array if the trash = true;
+      // listArray = listArray.filter (function(obj) {
+      //   return obj.trash !== true;
+      // });
```
Comment (prolajumokeoni) [inline src/index.js]: - [x] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @AbbyNyakara, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] Kindly write good and meaningful commit messages, your commits should describe changes  in a clear and concise manner this will help keep track of changes in your repository 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (AbbyNyakara) [reply src/index.js]: @prolajumokeoni , I have removed the commented code. Please review again.

**Review by julie-ify — APPROVED** [review]
Hi @AbbyNyakara,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**